pkg/workflows/wasm/runner.go

```diff
@@ -163,7 +163,7 @@ func (r *Runner) handleComputeRequest(factory *sdk.WorkflowSpecFactory, id strin
 		}
 	}
 
-	sdk := r.sdkFactory(drc)
+	sdk := r.sdkFactory(drc, WithRequestID(id), WithRequestMetaData(&creq.Metadata))
```

Comment:
id + requestMetadata aren't really options -- if they get omitted, then it'll be a misuse of the API. Wdyt about not having these as options and as having them as simple args?